src/app/advice-generator/advice-generator.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <div class="container__main-component">Advice detail</div>
```

Comment:
This should be in plural.

```suggestion
  <div class="container__main-component">Advice details</div>
```